src/coreComponents/physicsSolvers/multiphysics/MultiphasePoromechanicsSolver.cpp

```diff
@@ -210,14 +210,6 @@ void MultiphasePoromechanicsSolver::assembleSystem( real64 const time_n,
                                    dofManager,
                                    localMatrix,
                                    localRhs );
-
-  // Cell-based contribution
-  // TODO: move in PoroelasticKernels
-  m_flowSolver->assembleVolumeBalanceTerms( domain,
-                                            dofManager,
-                                            localMatrix,
-                                            localRhs );
-
```

Comment:
I don't see the deletion of the actual function anywhere?

Reply:
it's still used for flow only simulations.